haskell-src/Concordium/Types/Queries.hs

```diff
@@ -599,6 +600,14 @@ data PendingUpdateEffect
     | -- |Updates to time parameters for chain parameters version 1.
       PUETimeParameters !(TimeParameters 'ChainParametersV1)
 
+$( deriveJSON
```

Comment:
Describe what the intended format looks like, i.e., "updateType" as the tag, and "contents" as the payload.